src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2040,6 +2040,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="SuggestedRedirects" ItemName="SuggestedBindingRedirects"/>
       <Output TaskParameter="FilesWritten" ItemName="FileWrites"/>
       <Output TaskParameter="DependsOnSystemRuntime" PropertyName="DependsOnSystemRuntime"/>
+      <Output TaskParameter="DependsOnNETStandard" PropertyName="DependsOnNETStandard"/>
```

Comment:
The property name `DependsOnNETStandard` is also [used by the Microsoft.NET.Build.Extensions targets](https://github.com/dotnet/sdk/blob/d9955fcf3440f8596f4e63aa37a8dbcc477d97f4/src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets#L45).  Should we choose a different name here, or let them overlap?  I believe that if both the extensions target and RAR are writing to this property, they would agree on its value.  Thoughts? @ericstj

Reply:
Indeed they would agree on the value.  The extensions write to it earlier (honoring user specified value) and consume the value immediately.  RAR writes to it later ignoring a previously set value.  I don't think its a problem that they both use it, more of a style thing.

IMHO these should be `_` properties anyway since MSBuild isn't honoring a value previously set by the user (like we do in extensions for DependsOnNETStandard).